# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/add_sinf_cosf.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- uClibc-0.9.27/libm/s_sin.c.orig	2005-01-28 10:47:40.000000000 -0800
+++ uClibc-0.9.27/libm/s_sin.c	2005-01-28 10:55:01.000000000 -0800
@@ -80,3 +80,17 @@
 	    }
 	}
 }
+
+#ifdef __STDC__
+        float sinf(float x)
+#else   
+        float sinf(x)
+        float x;
+#endif
+{
+        double y;
+        y=x;
+        y=sin(y);
+        return (float)y;
+}
+
--- uClibc-0.9.27/libm/s_cos.c.orig	2005-01-28 10:44:30.000000000 -0800
+++ uClibc-0.9.27/libm/s_cos.c	2005-01-28 10:52:15.000000000 -0800
@@ -80,3 +80,16 @@
 	    }
 	}
 }
+
+#ifdef __STDC__
+	float cosf(float x)
+#else
+	float cosf(x)
+	float x;
+#endif
+{
+	double y;
+	y=x;
+	y=cos(y);
+	return (float)y;
+}
